config/sys.config

```diff
@@ -127,6 +127,11 @@
                ]}
              ]}
            ]}
+      ]},
+      {traces, [
+        {{lager_file_backend, "log/epoch_pow_cuckoo.log"},
+         [{module, aeminer_pow_cuckoo}],
```

Comment:
Could you please confirm that this is possible with OTP 21+ logger too?

Reply:
I see the log messages can have metadata, and there are filters which can use the metadata so the same should be possible with OTP 21+ logger.